build.ps1

```diff
@@ -31,12 +31,16 @@ Param(
     [string[]]$ScriptArgs
 )
 
-$CakeVersion = "0.19.3"
+$CakeVersion = "0.22.1"
 $DotNetChannel = "preview";
-$DotNetVersion = "1.0.1";
-$DotNetInstallerUri = "https://raw.githubusercontent.com/dotnet/cli/v1.0.1/scripts/obtain/dotnet-install.ps1";
+$DotNetVersion = "1.1.4";
```

Comment:
This file, `build.ps1` is a direct copy of the Cake 0.22.1 release version except for this one line that I updated from `1.0.4` to `1.1.4`. Everything else is as shipped.